testProjectCustomProtoDir/src/main/protocol buffers/more2.proto

```diff
@@ -0,0 +1,9 @@
+syntax = "proto3";
+
```

Comment:
nit: add comment here about the specially named "protocol buffers" directory, or rename it to something like "directory with space chars" to be more obvious about its purpose.